acme/acme/client_test.py

```diff
@@ -621,6 +622,26 @@ def test_requests_error_passthrough(self, mock_requests):
         self.assertRaises(requests.exceptions.RequestException,
                           self.net._send_request, 'GET', 'uri')
 
+    def test_urllib_error(self):
+        try:
+            # pylint: disable=protected-access
+            self.net._send_request('GET', "http://localhost:19123/nonexistent.txt")
+
+        except ValueError as y:
+            if "linux" in sys.platform:
+                self.assertEqual("Requesting localhost/nonexistent: "
+                                 "[Errno 111] Connection refused", str(y))
+            else: #pragma: no cover
+                self.assertEqual("Requesting localhost/nonexistent: "
+                                 "[Errno 61] Connection refused", str(y))
+
+        except requests.exceptions.ConnectionError as z: #pragma: no cover
```

Comment:
It looks like, on python3, hitting this error will print `('Connection aborted.', error(111, 'Connection refused')`, and you've made the test check that behavior. But I think that behavior is wrong. We'd like to make this print the same error on python3 that it prints on other Python versions. I think the best thing to do here is install python3 and figure out why you're not getting the expected results there.